voting/orbs/Elections/processing_vote.go

```diff
@@ -185,6 +190,12 @@ func _getStakeAtElection(ethAddr [20]byte) uint64 {
 	return ((*stake).Div(*stake, ETHEREUM_STAKE_FACTOR)).Uint64()
 }
 
+func _getLockedStakeAtElection(ethAddr [20]byte) uint64 {
```

Comment:
same function as `_getStakeAtElection` with the contract (address and ABI) and method names replaced. consider passing them as params to a unified function...

Reply:
i prefer to make this function separate. this is all temporary code for 4-6 months.